lib/location/location_core.c

```diff
@@ -518,9 +518,9 @@ static void location_core_timeout_work_fn(struct k_work *work)
 	location_core_event_cb_timeout();
 }
 
-void location_core_timer_start(uint16_t timeout)
+void location_core_timer_start(int32_t timeout)
 {
-	if (timeout > 0) {
+	if (timeout != SYS_FOREVER_MS || timeout > 0) {
```

Comment:
Do you mean `(timeout != SYS_FOREVER_MS) && (timeout > 0)` ?

Reply:
Absolutely